samples/snippets/gemini_model_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_gemini_text_generator_model():
+    # Determine project id, in this case prefer the one set in the environment
+    # variable GOOGLE_CLOUD_PROJECT (if any)
+    import os
+
+    PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT", "bigframes-dev")
+    REGION = "us"
+    CONN_NAME = "bigframes-default-connection"
+
+    # [START bigquery_dataframes_gemini_model]
+    from bigframes.ml.llm import GeminiTextGenerator
+    import bigframes.pandas as bpd
+
+    # Create the Gemini LLM model
+    session = bpd.get_global_session()
+    connection = f"{PROJECT_ID}.{REGION}.{CONN_NAME}"
+    model = GeminiTextGenerator(session=session, connection_name=connection)
+
+    df_api = bpd.read_csv("gs://cloud-samples-data/vertex-ai/bigframe/df.csv")
```

Comment:
not related to this PR but "bigframe" sounds odd on a GA public documentation, we should change it to "bigframes" and update in our code samples

Reply:
Ack. Let's file a bug for it.